### PR TITLE
Repair unqualified zlog module names

### PR DESCRIPTION
**Pull request by ffoulkes**
- Added the prefix BF_MOD_ to the unqualified zlog module name KRNLMON, for consistency with the other module names and to avoid contaminating the global namespace with unqualified symbols.

- Eliminated the unused zlog module names OVSP4RT and INFRAP4D.

Signed-off-by: Derek G Foster <derek.foster@intel.com>

### CODE REVIEW

**Comment by ffoulkes** [comment]
I eliminated the OVSP4RT and INFRAP4D symbols (instead of renaming them) on the _No Dead Code_ principle.

**Review by saynb — APPROVED** [review]
LGTM. But I would wait for approval from one more reviewer

**Comment by ffoulkes** [comment]
This change won't become breaking until is has been propagated through _target-utils_ to _p4-dpdk-target_.

At this point, _krnlmon_ will need to be updated to use the renamed symbol, and the change will need to be propagated to _ipdk-io/networking-recipe._

I can mitigate the effect to some extent by adding CMake logic to check for the existence of old and new version of the symbol.

File: include/target-sys/bf_sal/bf_sys_log.h
Comment (weiqiand) [inline include/target-sys/bf_sal/bf_sys_log.h:83]: Those modules are added by Nupur. Just forward to Nupur to review it for OVS side.

Comment (ffoulkes) [reply include/target-sys/bf_sal/bf_sys_log.h:83]: I believe Nupur is still on marital leave. I'm TL on the project.

Comment (ffoulkes) [reply include/target-sys/bf_sal/bf_sys_log.h:83]: Come to think of it, I could mitigate the KRNLMON change by defining `KRNLMON` as a synonym for the new symbol and marking the old name as deprecated.

Comment (ffoulkes) [reply include/target-sys/bf_sal/bf_sys_log.h:83]: ... which I have now done.

**Pull request closed without merging**
